### DISCUSSION TITLE
switching between organisation and personal settings impossible

### DISCUSSION THREAD

**Discussion by brunt82** [dani-garcia/vaultwarden#4766] [Q&A] [answered]
Hi,

maybe you already know. But with the new UI design it is impossible to switch between the settings of the organizations and the personal vault.
You only get to the settings page when you create the organization. After that there is no longer any option via the menu.
The reason for this is certainly the change in the URL from folder-based to parameter-based:
/organizations/39fb3115-f12c-4825-9eec-6b1cfdb9be6b/vault 
vs.
/vault?organizationId=39fb3115-f12c-4825-9eec-6b1cfdb9be6b

If you call up the "old" folder-based URL, you get to the org settings.

Regards,
brunt

### Deployment environment

* vaultwarden version: 1.31.0

* Install method: docker

* Clients used:  web-vault

* Reverse proxy and version: caddy:latest

* MySQL/MariaDB or PostgreSQL version: 11.4.2-MariaDB-ubu2404

* Other relevant details:

**Comment by illmouse** [comment]
Can confirm on a fresh installation.

**Comment by crhiles** [comment]
What happens when you go to the admin console?
![admin console](https://github.com/user-attachments/assets/d0f42b0d-e011-4eed-8286-04707bf683cc)

**Comment by brunt82** [comment]
Hi,

I don't see an "Admin Console" button. However, I can of course open the Admin Console if necessary (~/admin/).

But it's about the policies that I want to set for the organization. In the "old" UI, I had a menu item called "Organizations". This is missing in the new UI:
![grafik](https://github.com/user-attachments/assets/162ccaea-7d3c-4ffc-94ab-a1d8037fca5d)
vs.
![grafik](https://github.com/user-attachments/assets/d71b74b5-81aa-4af4-931d-938a11aca35e)

Best regards,
brunt

**Comment by illmouse** [comment]
My bad. I've missed this 9-dots icon which has "Password manager" and "Admin console" options. Admin console opens organization section. Could be not very fluent UX though :)

**Reply by stefan0xC** [reply, marked as answer]
See [this blog article](https://bitwarden.com/blog/bitwarden-design-updating-the-navigation-in-the-web-app/) for more background information about this change.

The ([erroneously](https://www.reddit.com/r/Bitwarden/comments/1e6zsm9/answers_whats_going_on_with_the_latest_release/) released) `web-v2024.7.0` version of the web-vault seems to make switching to the admin console more accessible again:
![Screenshot 2024-07-22 at 08-59-55 Vaults Vaultwarden Web](https://github.com/user-attachments/assets/2794cb73-664e-4685-812d-d759bdb03e3c)

So this will hopefully be not an issue in future versions anymore.

**Reply by brunt82** [reply]
The link to the blog helps to understand the UI changes and leads me to the admin-console button. I'm sorry, I ignored the 9-dot menu and therefore didn't see that menu.
